api/osrd_infra/views/study.py

```diff
@@ -0,0 +1,206 @@
+from django.db.models import Q
+from django.http import HttpResponse
+from PIL import Image
+from rest_framework import filters, generics, mixins, status
+from rest_framework.decorators import action
+from rest_framework.generics import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.viewsets import GenericViewSet
+
+from osrd_infra.models import Project, Scenario, Study, Timetable
+from osrd_infra.schemas.study import StudyState, StudyType
+from osrd_infra.serializers import (
+    ProjectSerializer,
+    ScenarioSerializer,
+    StudySerializer,
+)
+from osrd_infra.views.pagination import CustomPageNumberPagination
+
+
+class ScenarioView(
+    mixins.CreateModelMixin,
+    mixins.RetrieveModelMixin,
+    mixins.DestroyModelMixin,
+    mixins.UpdateModelMixin,
+    mixins.ListModelMixin,
+    GenericViewSet,
+):
+    serializer_class = ScenarioSerializer
+    filter_backends = [filters.OrderingFilter]
+    ordering_fields = ["name", "creation_date", "last_modification"]
+    pagination_class = CustomPageNumberPagination
+
+    def get_queryset(self):
+        """this function will disappear to be replaced by the generic search"""
```

Comment:
you can add a link to the related github ticket, it is not really documentation but more a comment to start with `#`